paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/definitions/MultibancoDefinition.kt

```diff
@@ -0,0 +1,36 @@
+package com.stripe.android.lpmfoundations.paymentmethod.definitions
+
+import com.stripe.android.lpmfoundations.luxe.SupportedPaymentMethod
+import com.stripe.android.lpmfoundations.paymentmethod.AddPaymentMethodRequirement
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodDefinition
+import com.stripe.android.lpmfoundations.paymentmethod.PaymentMethodMetadata
+import com.stripe.android.lpmfoundations.paymentmethod.UiDefinitionFactory
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.ui.core.R
+import com.stripe.android.ui.core.elements.SharedDataSpec
+
+internal object MultibancoDefinition : PaymentMethodDefinition {
+    override val type: PaymentMethod.Type = PaymentMethod.Type.Multibanco
+
+    override val supportedAsSavedPaymentMethod: Boolean = false
+
+    override fun requirementsToBeUsedAsNewPaymentMethod(
+        hasIntentToSetup: Boolean
+    ): Set<AddPaymentMethodRequirement> = setOf(
+        AddPaymentMethodRequirement.UnsupportedForSetup,
+        AddPaymentMethodRequirement.MerchantSupportsDelayedPaymentMethods,
+    )
+
+    override fun requiresMandate(metadata: PaymentMethodMetadata): Boolean = false
+
+    override fun uiDefinitionFactory(): UiDefinitionFactory = MultibancoUiDefinitionFactory
+}
+
+private object MultibancoUiDefinitionFactory : UiDefinitionFactory.RequiresSharedDataSpec {
+    override fun createSupportedPaymentMethod(sharedDataSpec: SharedDataSpec) = SupportedPaymentMethod(
+        paymentMethodDefinition = MultibancoDefinition,
+        sharedDataSpec = sharedDataSpec,
+        displayNameResource = R.string.stripe_paymentsheet_payment_method_multibanco,
+        iconResource = R.drawable.stripe_ic_paymentsheet_pm_alma,
```

Comment:
This is intentionally wrong; still trying to figure out where we get this asset from